### ISSUE TITLE
C++23 z/Z suffix (for clang/gcc modes)

### ISSUE THREAD

**Issue by ilobilo** [microsoft/vscode-cpptools#10190]
### Feature Request

C++23 adds support for z/Z and zu/ZU suffixes to integer literals (signed size_t and size_t)
I can enable C++23 from vscode settings but if I try to use z suffix, vscode shows this error:
```
user-defined literal operator not found
```
for example:
```c++
auto a = -123z; // signed size_t
auto b = 321zu; // size_t
```

**Comment by Zingam** [comment]
C++23 is not supported yet.

**Comment by sean-mcmanus** [comment]
@ilobilo  Yeah, same issue as https://github.com/microsoft/vscode-cpptools/issues/9878 . You can check https://en.cppreference.com/w/cpp/23 for the progress, in particular the "EDG eccp" column will need to add support for [Literal suffix](https://en.cppreference.com/w/cpp/language/integer_literal) for (signed) [size_t](https://en.cppreference.com/w/cpp/types/size_t) and then it may some time for us to get those changes integrated/shipped.

**Comment by michelleangela** [comment]
This is resolved in https://github.com/microsoft/vscode-cpptools/releases/tag/1.16.0.

**Comment by michelleangela** [comment]
Fix is now available in [release version 1.16.2](https://github.com/microsoft/vscode-cpptools/releases/tag/v1.16.2).
